### DISCUSSION TITLE
How to enable wire:navigate in laravel folio with or without volt?

### DISCUSSION THREAD

**Discussion by discoverlance-com** [livewire/livewire#6080] [1. Help] [answered]
I have a simple application which I am using laravel folio with livewire already installed but I realised that without adding a livewire component or volt component, `wire:navigate` does not work.

I created a livewire layout with `php artisan livewire:layout` and each folio page extended it but `wire:navigate` basically did not work till I tried to introduce volt in some parts of the page with anonymous component.

```blade
{{ -- This does not work (page loads like normal laravel page routing) --}}
<x-layouts.app>
  <h1>My Page</h1>
  <a wire:navigate href="/about">About</a>

</x-layouts.app>


{{ -- This works (Page loads client side with livewire) --}}
<x-layouts.app>
  <h1>My Page</h1>

  @volt
       <a wire:navigate href="/about">About</a>
  @endvolt
</x-layouts.app>


{{ -- This also works (Page loads client side with livewire) --}}
<x-layouts.app>
  @volt
      <h1>My Page</h1>

  
       <a wire:navigate href="/about">About</a>
  @endvolt
</x-layouts.app>
```

Is there a way to ensure that if I have livewire installed, I can use `wire:navigate` without having to necessarily use volt on my page or use single class based livewire page. I am referring to being able to use this in terms of using Folio directly with `wire:navigate` or even in blade. Or is this a feature that's only possible with livewire single class pages or volt present in the page.

I am assuming that it's not working because, by default, livewire is not shipped by default with the folio pages or a blade view even if have it using or extending the livewire (`php artisan livewire:layout` ) layout till you use livewire in it like a volt component so it could be the reason it's not working by default.

**Comment by joshhanley** [comment]
@discoverlance-com currently, I don't think there is a way to use navigate without a Livewire component on the page.

**Comment by robsontenorio** [comment]
A dirty workaround is on main layout just put a empty livewire component. 

Let me know if there is a more elegant solution.

**Comment by joshhanley** [comment, marked as answer]
We're in the process of investigating all wire:navigate issues and the reason is doesn't work is because Livewire's scripts aren't injected automatically if there is no Livewire component. So you just need to ensure you have `@livewireScripts` in your layout file for the pages that don't have Livewire on them. Because of that, I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
